.github/workflows/gradle.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           java-version: ${{ matrix.java_version }}
 
       - name: Test with Gradle
-        run: ./gradlew --parallel --continue javadoc spotlessCheck check
+        run: ./gradlew --parallel --continue javadoc spotlessCheck check --info
```

Comment:
Is it intended to keep the `--info` flag? The build log is quite large right now.